ts/sveltelib/handler-list.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright: Ankitects Pty Ltd and contributors
+// License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+
+import type { Readable, Writable } from "svelte/store";
+import { writable } from "svelte/store";
+
+import type { Callback } from "../lib/typing";
+
+type Handler<T> = (args: T) => Promise<void>;
+
+interface HandlerAccess<T> {
+    callback: Handler<T>;
+    clear(): void;
+}
+
+class TriggerItem<T> {
+    #active: Writable<boolean>;
```

Comment:
Hm, I think Typescript compiled it into something compatible. Shouldn't have browserslist also complained otherwise?

Reply:
Ah, you're right - looks like TypeScript is transpiling it for us, which is great. Browserslist is a useful tool, but it doesn't have 100% coverage, so it can sometimes pass even when we have code that is incompatible.